modules/install-sno-requirements-for-installing-single-node-openshift.adoc

```diff
@@ -0,0 +1,34 @@
+// This is included in the following assemblies:
+//
+// installing_sno/install-sno-preparing-to-install-sno.adoc
+
+[id="requirements-for-installing-single-node-openshift_{context}"]
+= Requirements for installing single node OpenShift
+
+Single node OpenShift alleviates some of the requirements of high availability and large scale clusters. However, you must address the following requirements:
+
+* *Administration node:* You must have an administration node or laptop to access link:https://cloud.redhat.com/openshift/assisted-installer/clusters[Install OpenShift with the Assisted Installer], to create the cluster, to create the USB boot drive, and to monitor the installation.
+
+* *Production-grade server:* A single node cluster requires a server with sufficient resources to run {product-title} services and a production workload.
```

Comment:
"single node cluster" is slightly oxymoronic, but +100 for not using words that suggest it is a different product.  Keep it, I'm just highlighting that this is the kind of language to use.